wordpress-mobile/WordPress-Android
# Feature/added additional domain credit tracking

We are adding some extra tracking to domain credit redemption:

- When CTA is displayed.
- When user inputs query at domain suggestion screen (only once).
- When a user selects a domain name and proceeds to contact form.

To test:
- Switch to a site with unredeemed domain credit.
- Put breakpoints near each tracking call, and make sure they are hit when expected.

Update release notes:
